.github/workflows/push.yaml

```
@@ -0,0 +1,262 @@
+          zip -r app.ipa Payload
+        # This packages the Runner.app into an .ipa file
+
+      - uses: ncipollo/release-action@v1
+        with:
+          name: "Automated iOS Release"
+          artifacts: "app-release.ipa"
+          allowUpdates: "true"
+          generateReleaseNotes: false
+          tag: "automated"
+          body: |
+            This is an automated release, triggered by a recent push. 
```

yamllint

[error] trailing spaces

(trailing-spaces)